### ISSUE TITLE
Add client X/Y to Options class

### ISSUE THREAD

**Issue by joshlester** [manolo/gwtquery-gesture-plugin#7]
Hi Manuel,

Is it be possible to add client X/Y to the Options class? Currently it looks like only screen X/Y are available. Is there a way to calculate client X/Y?

PS. gQuery is awesome, thanks for all your work.

**Comment by manolo** [comment]
Added client and page values. Updated SNAPSHOT, could you test if everything works for you.

**Comment by biancashouse** [comment]
The demo is broken ?

ios
-------
On an ipad in both Safari and Chrome the image just shakes and if you flick
the image up, for example, the image is somewhere off the screen, and
eventually comes back.

i plugged into desktop Safari via USB and see a typeError undefined is not
an object...

android
----------
Also tried the demo on my nexus 7. Image is not shaking. pinch gesture
works, but the image is jumpy.
image can be flicked around the screen ok. flick up does cause a temporary
scroll, but that's just a html/css issue i think.

Rotate doesn't work on either device. Your button makes the image rotate,
but I was expecting to be able to rotate using 2 fingers ?

On my nexus 7, you cannot flick the image, after you have pinched in or out
to resize. In fact, even if i reload the demo in another tab, touch events
no longer work. In fact, even if I restart chrome, touch events not working
any more.

bottom line
---------------
i am not using the gesture plugin in my app right now. i was thinking the
flick/swipe gesture might be useful for something: Right now I have sliding
tabs that each have a draggable so the user can slide it in/out from the
side of the screen, but with your plugin I was wondering if i could remove
the button and instead use a swipe gesture.

Ian
